templates/etc/init.d/rpm/td-agent

```diff
@@ -144,7 +141,7 @@ if [ -n "${TD_AGENT_USER}" ]; then
   fi
   mkdir -p "$(dirname "${TD_AGENT_PID_FILE}")"
   chown -R "${TD_AGENT_USER}" "$(dirname "${TD_AGENT_PID_FILE}")"
-  START_STOP_DAEMON_ARGS="${START_STOP_DAEMON_ARGS} --user ${TD_AGENT_USER}"
+  START_STOP_DAEMON_ARGS="${START_STOP_DAEMON_ARGS} -u ${TD_AGENT_USER}"
```

Comment:
Is this included as a tidiness update, or is it required on SLES?

Reply:
This is needed because, unlike `daemon`, LSB's `start_daemon` only supports short options (`-u`, `-p`).